### PR TITLE
Fixed dim issue with SchNET implementation

### PR DESCRIPTION
**Pull request by wiederm**
## Description
THis PR fixes a bug in the padding of the embedding and interaction layer of SchNET.

## Status
- [x] Ready to go

### CODE REVIEW

File: modelforge/tests/helper_functinos.py
Comment (chrisiacovella) [inline modelforge/tests/helper_functinos.py:1]: typo in the name here "helper_functinos.py" should be "helper_functions.py"

File: modelforge/potential/schnet.py [hunk 1]
```diff
@@ -237,7 +244,7 @@ def forward(self, pairlist: Dict[str, torch.Tensor]) -> Dict[str, torch.Tensor]:
 
         idx_i, idx_j = atom_index12[0], atom_index12[1]
 
-        return x
+        return {"f_ij": f_ij, "idx_i": idx_i, "idx_j": idx_j, "rcut_ij": rcut_ij}
 
 
 class LighningSchnet(Schnet, LighningModuleMixin):
```
Comment (chrisiacovella) [inline modelforge/potential/schnet.py:250]: Two things here:

- We should probably consistent of `Schnet` vs `SchNet`
- minor typo in Lightning (forgot the t).

File: modelforge/potential/schnet.py [hunk 2]
```diff
@@ -162,20 +180,15 @@ def forward(
...
         return x
 
 
 class SchNetRepresentation(nn.Module):
     def __init__(
         self,
-        n_atom_basis: int,
-        n_filters: int,
-        n_interactions: int,
         cutoff: float = 5.0,
```
Comment (chrisiacovella) [inline modelforge/potential/schnet.py:192]: It would be good here to specify the units (even without using the unit package, just annotate what units cutoff should be in).  Also I think the data we are loading in is nanometers, so this should be 0.5 (since 5.0 angstrom is used in schnet, right)?

**Review by chrisiacovella — CHANGES_REQUESTED** [review]
A few minor things at this point.  I still need to go through more thoroughly after I fix up my install on my machine.

File: modelforge/potential/schnet.py (hunk 2 above)
Comment (wiederm) [reply modelforge/potential/schnet.py:192]: that's great catch, thank you!

**Comment by chrisiacovella** [comment]
I'll note all the failures are coming from the changes to zenoto that break some of the functions I wrote (see #31 ). I have these updated/removed in a separated PR (#30 ), so those failing tests can be ignored for now.

**Review by chrisiacovella — APPROVED** [review]
Looking through this I'm not seeing any obvious issues.  I think we can merge now, that way we can get to a point to do more detailed tests/comparisons.
